stdlib/universe/moving_average.go

```diff
@@ -225,8 +221,6 @@ func (t *movingAverageTransformation) Finish(id execute.DatasetID, err error) {
 	t.d.Finish(err)
 }
 
-const movingAverageUnsortedTimeErr = "moving average found out-of-order times in time column"
-
 func (t *movingAverageTransformation) passThroughBool(ts *array.Int64, vs *array.Boolean, b execute.TableBuilder, bj int) error {
```

Comment:
I would say  that you don't  need this function anymore right? At this point you can simply remove the first `N - 1` records and append the rest of the column.

Reply:
(Also for string and time)